supermarq-benchmarks/supermarq/qcvv/base_experiment_test.py

```diff
@@ -163,6 +163,108 @@ def test_results_init(
     assert results.samples == abc_experiment.samples
     assert results.num_circuits == 10
     assert results.num_qubits == 2
+    assert results.qubits == (cirq.q(0), cirq.q(1))
+
+
+def test_results_getitem(
+    abc_experiment: ExampleExperiment,
+) -> None:
+    q0, q1 = abc_experiment.qubits
+
+    results = abc_experiment.run_with_callable(lambda c: {"00": 10, "01": 10})
```

Comment:
Does the experiment expect counts (`"00":10, "01": 10`) and then convert it into a probability distribution (`{"00": 0.5, "01": 0.5}`)?

Reply:
it will accept either probabilities or counts (and in the latter case automatically converts to a probability distribution). But that's not what's being tested here so tbh not sure why i did it this way..

updated to instantiate the results object directly instead of this weird use of `run_with_callable`